google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/JsonToProtoMessageTest.java

```diff
@@ -108,6 +117,21 @@
           .put(RepeatedObject.getDescriptor(), "object")
           .build();
 
+  private static ImmutableMap<Descriptor, Message[]> BytesRepeatedCorrectProto =
```

Comment:
Could you add this case directly to AllTypesToDebugMessageTest?

Reply:
I've been working on adding it in like that but the issue is it runs into an error with multiple entries with the same key because of the existing BYTES related test that covers the String path. I've been messing with adding it in and it's resulting in some very messy nesting and checking against multiple possible outcomes that obscures what is being done and how it fails. I can probably make it work but I don't know if it is the best option.

Reply:
I was messing with that for a while but I think your comment below actually kind of solves the issue I was having with it. I just added in the positive test to the AllRepeatedTypesToDebugmessageTest and it works now. I will see if the negative test can also be added in now.